nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -580,14 +619,24 @@ export function EPCEdit(props: Props) {
   const ctx = useContext(GatewayContext);
 
   const handleEPCChange = (key: string, val) => {
-    setEPCConfig({...EPCConfig, [key]: val});
+    if (key === 'nat_enabled' && val) {
```

Comment:
we seem to be setting [key]: val regardless, is val check really necessary here.